src/Firebase/firebaseConfig.example.json

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "service_account",
+  "project_id": "PROJECT_ID",
+  "private_key_id": "YOUR_PRIVATE_KEY_ID",
+  "private_key": 'YOUR_PRIVATE_KEY',
+  "client_email": "YOUR_EMAIL",
+  "client_id": "YOUR_ID",
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://oauth2.googleapis.com/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/firebase-adminsdk-ydkke%40testproject-258d3.iam.gserviceaccount.com",
+  "universe_domain": "googleapis.com"
+}
```

Comment:
ENV

Reply:
i have changed it but don't get the difference tryed to search some info about that still not getting why env is better then json 😅